apps/ledger-live-mobile/src/components/CustomImage/ImageProcessor.tsx

```diff
@@ -1,7 +1,7 @@
 import { Flex } from "@ledgerhq/native-ui";
 import React from "react";
 import { WebView } from "react-native-webview";
-import { WebViewErrorEvent, WebViewMessageEvent } from "react-native-webview/lib/WebViewTypes";
+import { WebViewErrorEvent, WebViewMessageEvent } from "react-native-webview/src/WebViewTypes";
```

Comment:
same remark as Gaetan on the other file: importing from src ?

Reply:
As discussed, I removed the console.log associated so we don't need to use `WebViewErrorEvent` anymore and `WebViewMessageEvent` is imported from `react-native-webview` now